toolkit/scripts/pkggen.mk

```diff
@@ -125,12 +124,13 @@ ifeq ($(STOP_ON_FETCH_FAIL),y)
 graphpkgfetcher_extra_flags += --stop-on-failure
 endif
 
-$(cached_file): $(graph_file) $(go-graphpkgfetcher) $(chroot_worker) $(pkggen_local_repo) $(depend_REPO_LIST) $(REPO_LIST) $(call shell_real_build_only, find $(CACHED_RPMS_DIR)/) $(pkggen_rpms) $(TOOLCHAIN_MANIFEST)
```

Comment:
Is `pkggen_rpms` even still used anywhere?

Reply:
Hmm, looks like not... I feel like `$(RPMS_DIR):` should maybe depend on it though? If we manually add an rpm, we should retrigger an image build since one of its component RPMs may have changed.

I'm going to leave it for now and look at that in the future.